extensions/http-curl/processors/InvokeHTTP.cpp

```diff
@@ -149,6 +149,13 @@ const core::Relationship InvokeHTTP::RelFailure("failure",
     "The original FlowFile will be routed on any type of connection failure, "
     "timeout or general exception. It will have new attributes detailing the request.");
 
+
+const core::OutputAttribute InvokeHTTP::StatusCode{"invokehttp.status.code", { Success, RelResponse, RelRetry, RelNoRetry }, "The status code that is returned"};
```

Comment:
can we use the `STATUS_CODE` and other constants from `InvokeHTTP.h`?

Reply:
we could also work in the other direction in a subsequent PR and use these instead of constant strings

Reply:
good idea, fixed in b520a96bc29abfb4019ae92cd167ce806b06a4c4